build-a-standalone-application.md

```diff
@@ -24,6 +24,26 @@ We use [`humbug/box`](https://github.com/box-project/box) to provide fast applic
 
 Please check the box documentation to understand all options: [github.com/box-project/box/blob/master/doc/configuration.md](https://github.com/box-project/box/blob/master/doc/configuration.md).
 
+<a name="distribute-via-packagist"></a>
+## Distribute via Packagist
+
+To distribute your application via [Packagist](https://packagist.org) you will need to make some changes to your `composer.json` & `box.json` files.
+
+- In your `composer.json` file you will need to move the `laravel-zero/framework` dependency from `require` to `require-dev`.
+
+- In your `box.json` file you should add `"exclude-dev-files": false,`
+
+Now you will need to build your application again with:
+```bash
+php <your-app-name> app:build <your-build-name>
+```
+
+And you are ready to distribute your package via [Packagist](https://packagist.org)  and install it by running
```

Comment:
It's missing here the `bin: build/your-build-name`.

Reply:
Oh Yeah that's right!